cmd/generate/generate.go

```diff
@@ -101,24 +102,10 @@ type Schema struct {
 	OAuthAuthorizeTokenList           oauthapi.OAuthAuthorizeTokenList
 	OAuthClientList                   oauthapi.OAuthClientList
 	OAuthClientAuthorizationList      oauthapi.OAuthClientAuthorizationList
-	ClusterPolicy                     authapi.ClusterPolicy
-	ClusterPolicyList                 authapi.ClusterPolicyList
```

Comment:
we need the openshift RBAC objects to remain - but add them in addition to the kubernetes RBAC changes